cves/CVE-2013-0870.yml

```diff
@@ -150,10 +161,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: n/a
+  fix: false
+  fix_answer: n/a
```

Comment:
`n/a` is not a sufficient response. If you could not find any unit tests in nearby directories or in the fix, then please explicitly say that.

Reply:
ah okay, i wasn't sure how to depict that i couldn't find anything